apps/src/templates/teacherNavigation/TeacherNavigationBar.tsx

```diff
@@ -41,6 +42,102 @@ const TeacherNavigationBar: React.FunctionComponent = () => {
     }
   }, [sections, selectedSectionId]);
 
+  const courseContent = (
+    <>
+      <Typography
+        semanticTag={'h2'}
+        visualAppearance={'overline-two'}
+        className={styles.sectionHeader}
+      >
+        {i18n.courseContent()}
+      </Typography>
+      <TeacherNavigationSidebarOption
+        icon={'desktop'}
+        optionTitle={i18n.course()}
+        isSelected={false}
+      />
+      <TeacherNavigationSidebarOption
+        icon={'folder-open'}
+        optionTitle={i18n.lessonMaterials()}
+        isSelected={false}
+      />
+      <TeacherNavigationSidebarOption
+        icon={'file-lines'}
+        optionTitle={i18n.lessonPlans()}
+        isSelected={false}
+      />
+      <TeacherNavigationSidebarOption
+        icon={'presentation-screen'}
+        optionTitle={i18n.slideDecks()}
+        isSelected={false}
+      />
+      <TeacherNavigationSidebarOption
+        icon={'calendar'}
+        optionTitle={i18n.calendar()}
+        isSelected={false}
+      />
+    </>
+  );
+
+  const performanceContent = (
+    <>
+      <Typography
```

Comment:
```suggestion
  const performanceContent = [
      <Typography
```
I think we can use an array `[]` instead of empty tag `<>`

Reply:
Good idea - thanks for pointing this out as an option - I can see how it might be helpful when it comes to identifying the "selected" option too.